### PR TITLE
Symfony 3 support

### PR DESCRIPTION
**Pull request by lucasvanlierop**
I'd like to use this bundle in a Symfony 3 project, this PR will be an attempt to support Symfony 3. I need to time box it however so I can't guarantee it will work.

### CODE REVIEW

File: src/Monolog/SamlAuthenticationLogger.php [hunk 1]
```diff
@@ -26,6 +26,7 @@
  * Decorates a PSR logger and adds information pertaining to a SAML request procedure to each message's context.
  *
  * @SuppressWarnings(PHPMD.TooManyMethods)
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
```
Comment (lucasvanlierop) [inline src/Monolog/SamlAuthenticationLogger.php]: Somehow this 'works on my machine' but is ignored by PHPMD running on Travis

**Comment by lucasvanlierop** [comment]
@thijskh I've been using a fork of this project in a Symfony 3 project for a while now. Seems to work perfect. After a small change in the PHPMD ignore config the build now succeeds on PHP 5.6 and 7.0 (but not yet on 7.1). Is there a timeframe to drop support for older PHP versions?

**Comment by thijskh** [comment]
I would say that it should be fine to require 5.6 as the minimum. But maybe I'm overlooking something.

@jorissteyn @MKodde @pmeulen Do you see any objections to drop 5.4 and 5.5 in stepup-saml-bundle? And have any other comments that should prevent merging this?

**Comment by jorissteyn** [comment]
Hi Lucas! @lucasvanlierop I've relaxed the version constraint in composer.json to allow for both symfony2.7 (which we use in production) and symfony3. I've also removed the lockfile so Travis can build on all versions of PHP we want it to build. I've tagged this version 2.9.0. Thanks for the effort!

**Comment by lucasvanlierop** [comment]
@jorissteyn Hi! Thanks for picking this up! We're going to test it :-)

**Pull request merged**